hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,7 @@
+RELEASE_TYPE: patch
+
+This release adds database support for :ref:`targeted property-based testing <targeted-search>`,
+so the best examples based on the targeting will be saved and reused between runs.
+This is mostly laying groundwork for future features in this area, but
+will also make targeted property based tests more useful during development,
```

Comment:
nit (I don't know if it makes a difference, but we might as well be consistent!)

```suggestion
will also make targeted property-based tests more useful during development,
```

Reply:
property-based testing is definitely the correct hyphenation, thanks!

Reply:
You've changed it in other places, but not here :stuck_out_tongue_closed_eyes: 

I'd also consider adding something like `git grep -l 'property based' | xargs sed -i 's/property based/property-based/g'` to our formatting task, to catch and auto-fix any new cases.

Reply:
oh, right, because I searched for "property based testing". Oops!